sdk/redis/arm-rediscache/recordings/node/redis_test/recording_redis_create_test.js

```diff
@@ -485,7 +485,7 @@ nock('https://management.azure.com:443', {"encodedQueryParams":true})
   .query(true)
-  .reply(201, {"id":"/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.Cache/Redis/myrediscachexxx111","location":"East US","name":"myrediscachexxx111","type":"Microsoft.Cache/Redis","tags":{},"properties":{"provisioningState":"Creating","redisVersion":"4.0.14","sku":{"name":"Premium","family":"P","capacity":1},"enableNonSslPort":true,"instances":[{"sslPort":15000,"nonSslPort":13000,"isMaster":false,"isPrimary":false},{"sslPort":15001,"nonSslPort":13001,"isMaster":false,"isPrimary":false},{"sslPort":15002,"nonSslPort":13002,"isMaster":false,"isPrimary":false},{"sslPort":15003,"nonSslPort":13003,"isMaster":false,"isPrimary":false}],"minimumTlsVersion":"1.2","publicNetworkAccess":"Enabled","redisConfiguration":{"maxmemory-policy":"allkeys-lru","maxclients":"7500","maxmemory-reserved":"200","maxfragmentationmemory-reserved":"300","maxmemory-delta":"200"},"accessKeys":{"primaryKey":"8ybKetYwvgeRyIrsdEeS53o3XI55bnWhyUw723b0Vc8=","secondaryKey":"zBZ3dUhgwU6vQgFbnSNhCrgxZ4QQncqhzYKobLOIdJ0="},"hostName":"myrediscachexxx111.redis.cache.windows.net","port":6379,"sslPort":6380,"subnetId":"/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.Network/virtualNetworks/networknamex/subnets/subnetworknamex","staticIP":"10.0.0.5","shardCount":2,"linkedServers":[]},"zones":["1"]}, [
+  .reply(201, {"id":"/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.Cache/Redis/myrediscachexxx111","location":"East US","name":"myrediscachexxx111","type":"Microsoft.Cache/Redis","tags":{},"properties":{"provisioningState":"Creating","redisVersion":"4.0.14","sku":{"name":"Premium","family":"P","capacity":1},"enableNonSslPort":true,"instances":[{"sslPort":15000,"nonSslPort":13000,"isMaster":false,"isPrimary":false},{"sslPort":15001,"nonSslPort":13001,"isMaster":false,"isPrimary":false},{"sslPort":15002,"nonSslPort":13002,"isMaster":false,"isPrimary":false},{"sslPort":15003,"nonSslPort":13003,"isMaster":false,"isPrimary":false}],"minimumTlsVersion":"1.2","publicNetworkAccess":"Enabled","redisConfiguration":{"maxmemory-policy":"allkeys-lru","maxclients":"7500","maxmemory-reserved":"200","maxfragmentationmemory-reserved":"300","maxmemory-delta":"200"},"accessKeys":{"primaryKey":"SecretPlaceholder","secondaryKey":"SecretPlaceholder"},"hostName":"myrediscachexxx111.redis.cache.windows.net","port":6379,"sslPort":6380,"subnetId":"/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.Network/virtualNetworks/networknamex/subnets/subnetworknamex","staticIP":"10.0.0.5","shardCount":2,"linkedServers":[]},"zones":["1"]}, [
```

Comment:
@HarshaNalluru using the new recorder this json replacement would be easier than using regex, right?

Reply:
yes